features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsPresenter.kt

```diff
@@ -147,10 +147,17 @@ class RoomDetailsPresenter @Inject constructor(
 
         val roomMemberDetailsState = roomMemberDetailsPresenter?.present()
 
+        val securityAndPrivacyPermissions = room.securityAndPrivacyPermissionsAsState(syncUpdateFlow.value)
+        val canShowSecurityAndPrivacy by remember {
+            derivedStateOf {
+                isKnockRequestsEnabled && roomType is RoomDetailsType.Room && securityAndPrivacyPermissions.value.hasAny
```

Comment:
Small neat, `securityAndPrivacyPermissions.value.hasAny` seems not to be live, I was on the room details screen when I got an admin of the room and the item "Security & Privacy" did not appear. I had to go back to the previous screen and come back to see it.

Reply:
Yes it's not live, because powerlevels are not live